SWHttpTrafficRecorder.m

```diff
@@ -75,11 +75,12 @@ - (BOOL)startRecordingAtPath:(NSString *)recordingPath forSessionConfiguration:(
             NSFileManager *fileManager = [NSFileManager defaultManager];
             if(![fileManager fileExistsAtPath:recordingPath]){
                 NSError *bError = nil;
-                [fileManager createDirectoryAtPath:recordingPath withIntermediateDirectories:YES attributes:nil error:&bError];
-                if(bError && error){
-                    *error = [NSError errorWithDomain:SWHttpTrafficRecorderErrorDomain code:SWHttpTrafficRecorderErrorPathFailedToCreate userInfo:@{NSLocalizedDescriptionKey: [NSString stringWithFormat:@"Path '%@' does not exist and error while creating it.", recordingPath]}];
+                if(![fileManager createDirectoryAtPath:recordingPath withIntermediateDirectories:YES attributes:nil error:&bError]){
```

Comment:
@JinlianWang, thanks. When `createDirectoryAtPath:withIntermediateDirectories:attributes:error:` returns `NO`, `bError` will be set and we’ll use it in our error’s `userInfo` dictionary with `NSUnderlyingErrorKey`.